specification/cognitiveservices/data-plane/WebSearch/v1.0/WebSearch.json

```diff
@@ -0,0 +1,1211 @@
+                "News",
+                "RelatedSearches",
+                "Videos",
+                "WebPageGroupings",
+                "Computation",
+                "TimeZone"
+              ],
+              "x-ms-enum": {
+                "name": "AnswerType",
+                "modelAsString": false
+              }
+            },
+            "collectionFormat": "csv"
+          },
+          {
+            "name": "safeSearch",
+            "x-ms-client-name": "SafeSearch",
+            "in": "query",
+            "description": "A filter used to filter adult content. Off: Return webpages with adult text, images, or videos. Moderate: Return webpages with adult text, but not adult images or videos. Strict: Do not return webpages with adult text, images, or videos. The default is Moderate. If the request comes from a market that Bing's adult policy requires that safeSearch is set to Strict, Bing ignores the safeSearch value and uses Strict. If you use the site: query operator, there is the chance that the response may contain adult content regardless of what the safeSearch query parameter is set to. Use site: only if you are aware of the content on the site and your scenario supports the possibility of adult content.",
+            "required": false,
+            "type": "string",
+            "enum": [
+              "Off",
+              "Moderate",
+              "Strict"
+            ],
+            "x-ms-enum": {
+              "name": "SafeSearch",
+              "modelAsString": false
+            }
+          },
+          {
+            "name": "setLang",
+            "x-ms-client-name": "SetLang",
+            "in": "query",
+            "description": "The language to use for user interface strings. Specify the language using the ISO 639-1 2-letter language code. For example, the language code for English is EN. The default is EN (English). Although optional, you should always specify the language. Typically, you set setLang to the same language specified by mkt unless the user wants the user interface strings displayed in a different language. This parameter and the Accept-Language header are mutually exclusive; do not specify both. A user interface string is a string that's used as a label in a user interface. There are few user interface strings in the JSON response objects. Also, any links to Bing.com properties in the response objects apply the specified language.",
+            "required": false,
+            "type": "string"
+          },
+          {
+            "name": "textDecorations",
+            "x-ms-client-name": "TextDecorations",
+            "in": "query",
+            "description": "A Boolean value that determines whether display strings should contain decoration markers such as hit highlighting characters. If true, the strings may include markers. The default is false. To specify whether to use Unicode characters or HTML tags as the markers, see the textFormat query parameter.",
```

Comment:
If this is true/false, it shouldn't be a string

Reply:
its a bool. Will update